_data/locales/en.yml

```diff
@@ -5,7 +5,7 @@ en:
     search: Search
     question: What Does Homebrew Do?
     what: Homebrew installs <a href="https://formulae.brew.sh/formula/" title="List of Homebrew packages">the stuff you need</a> that Apple (or your Linux system) didn’t.
-    how: Homebrew installs packages to their own directory and then symlinks their files into <code>/usr/local</code> (on macOS Intel).
+    how: Homebrew installs packages to their own directory and then symlinks their files into the Homebrew installation prefix.
```

Comment:
```suggestion
    how: Homebrew installs packages to their own directory and then symlinks their files into a common, shared path.
```

How about this?

Reply:
Still think it's nicer to have a specific example. The more generic it is the more useless this message becomes. Could make it `/opt/homebrew` as the most common case now and link out to others.

Reply:
```suggestion
    how: Homebrew installs packages to their own directory and then symlinks their files into the Homebrew installation prefix, such as /opt/homebrew.
```

Reply:
```suggestion
    how: Homebrew installs packages to their own directory and then symlinks their files into <code>/opt/homebrew</code> (on Apple Silicon).
```

if we're gonna stick closer to the original?

Reply:
Then we're back to the appearance of working only on one.

Reply:
```suggestion
    how: Homebrew installs packages to their own directory and then symlinks their files into <code title="Apple Silicon">/opt/homebrew</code> or <code title="Linux">/home/linuxbrew</code>.
```

And style it for a hover hint?

Reply:
Then that could be a compromise on the table thing, too (#939).

Reply:
That looks good to me, but it's `/home/linuxbrew/.linuxbrew` right?

Reply:
Technically, yes, but I figure Homebrew "owns" `/home/linuxbrew` so it's safe enough to shorten to just that for front-page documentations' sake.

Reply:
> Then we're back to the appearance of working only on one

I couldn't disagree more.

Let's not make this text longer. This site is intentionally a brief overview. We have way more comprehensive, precise documentation elsewhere.

At this point the only thing I'm really in favour of is replacing `/usr/local` with `/opt/homebrew` because we have/we will have more users using that than anything else.

Reply:
Also worth noting: the "`(or Linux)`" on the homepage is also a gentle nudge to the fact that most of our users use macOS and the Linux use-case is far more niche.